doc/user/content/self-hosted/_index.md

```diff
@@ -0,0 +1,262 @@
+
+To setup:
+
+1. Use Bottlerocket bootstrap container for LVM configuration.
+1. Configure volume group name as `instance-store-vg`
+
+
+
+#### Storage Configuration
+
+Once LVM is configured, set up the storage class:
+
+{{< tabs >}}
+{{< tab "OpenEBS" >}}
+
+If using OpenEBS, set up the storage class as follows:
+```yaml
+storage:
+  storageClass:
+    create: true
+    name: "openebs-lvm-instance-store-ext4"
+    provisioner: "local.csi.openebs.io"
+    parameters:
+      storage: "lvm"
+      fsType: "ext4"
+      volgroup: "instance-store-vg"
+```
+{{< /tab >}}
+{{< tab "Other Storage" >}}
+While OpenEBS is our recommended solution, you can use any storage provisioner that meets your performance requirements by overriding the provisioner and parameters values.
+
+For example, to use a different storage provider:
+
+```yaml
+storage:
+  storageClass:
+    create: true
+    name: "your-storage-class"
+    provisioner: "your.storage.provisioner"
+    parameters:
+      # Parameters specific to your chosen storage provisioner
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+## Installing the Chart
+
+1. If installing for the first time, create a namespace. The default
+   configuration uses the `materialize` namespace.
+
+   ```bash
+   kubectl create namespace materialize
+   ```
+
+1. Install the Materialize operator with the release name
+   `my-materialize-operator`
+
+      - To use the [default configuration](/self-hosted/configuration/):
+
+        ```shell
+        helm install my-materialize-operator materialize/misc/helm-charts/operator
+        ```
+
+      * To include custom configuration, you can:
+
+        - *Recommended:* Create or edita a YAML file (such as the sample
+          `values.yaml` file located in the
+          `materialize/misc/helm-charts/operator/` directory) that specifies the
+          values for the parameters and then install the chart with the `-f`
+          flag:
+
+          ```shell
+          helm install materialize/misc/helm-charts/operator/values.yaml  materialize/misc/helm-charts/operator
+          ```
+
+        - Specify each parameter using the `--set key=value[,key=value]`
+            argument to `helm install`. For example:
+
+            ```shell
+            helm install my-materialize-operator \
+              --set operator.image.tag=v1.0.0 \
+              materialize/materialize-operator
+            ```
+
+1. Verify the installation:
+
+    ```shell
+    kubectl get all -n materialize
+    ```
+
+## Deploying Materialize
+
+### Set up the metadata database
+
+
+### Set up blob storage
+
+
+### Deploy Materialize
+
+To deploy Materialize:
+
+1. Create a YAML file (e.g., `materialize.yaml`) with your Materialize blob
+   storage and metadata database configuration. The following example assumes
+   you have deployed PostgreSQL as the metadata database and MinIO as the blob
+   storage.
+
+
+   ```yaml
+   apiVersion: v1
+   kind: Namespace
+   metadata:
+     name: materialize-environment
```

Comment:
I know we had the discussion about scrubbing "Environment" -- not sure if we actually want to do that with our namespace and such.

Reply:
I vote we do... IMO it's worth removing the term entirely. From the discussion earlier this week, we can just reference "Materialize" or the Materialize custom resource when possible (and omit "environment"), and "Materialize instance" when we must refer to the resources associated with a specific `Materialize` CR.